guides/doc-Managing_Organizations_and_Locations/master.adoc

```diff
@@ -4,7 +4,12 @@ include::common/header.adoc[]
 
 = {ManagingOrganizationsLocationsDocTitle}
 
-ifndef::satellite[]
+// This guide is not ready for stable releases
+ifdef::HideDocument,satellite[]
+include::common/modules/snip_guide-not-ready.adoc[]
```

Comment:
Perhaps the text is not applicable for satellite since it's very upstream focused. Is that a problem?

Reply:
I doubt that many people look at "satellite" docs on theforeman.org. So I am fine either way; not building it because Satellite provided the content at other places is probably a good idea though.

Reply:
That was also my thought process, but I didn't know it was hidden for Satellite since it's provided elsewhere. Actually a good comment to have.